tests/workflows/basic_tool_calling_node_multi_tool/tests/test_workflow.py

```diff
@@ -32,7 +32,7 @@ def test_get_current_weather_workflow(vellum_adhoc_prompt_client, mock_uuid4_gen
     Test that the GetCurrentWeatherWorkflow returns the expected outputs.
     """
 
-    def generate_prompt_events(*args, **kwargs) -> Iterator[ExecutePromptEvent]:
+    def generate_prompt_events(*args, **kwargs) -> Iterator[ExecutePromptEvent]:  # noqa: U100
```

Comment:
I ended up getting rid of this: https://github.com/vellum-ai/vellum-python-sdks/pull/1814

feel free to rebase